src/istep/edbgIstep.C

```diff
@@ -0,0 +1,572 @@
+#include <edbgIstep.H>
+
+//Updated this table as per P10_IPL_Flow_v0.89.17. 
+//HB isteps need to revisit
+const edbgIPLTable::edbgIStep_t edbgIPLTable::cv_edbgIStepTable[] =
+{
+    /****************************************************************************/
+    /* !!! --- THIS LIST MUST BE IN ORDER THAT THEY'RE CALLED IN AN IPL --- !!! */
+    /****************************************************************************/
+
+    /****************************************************************************/
+    /* NOTE: The istep is executed by the self boot engine/the host code/       */
+    /*       the attached service processor depending upon the destination type.*/
+    /*       Options:                                                           */
+    /*       EDBG_ISTEP_HOST - The istep is performed by the host               */
+    /*       EDBG_ISTEP_SBE  - The istep is performed by the self boot engine   */
+    /*       EDBG_ISTEP_SP   - The istep is performed by the service processor  */
+    /*       EDBG_ISTEP_NOOP - The istep is No OP                               */
+    /*                                                                          */
+    /****************************************************************************/
+
+    /****************************************************************************/
+    /* Warning : The following constants are defined based on the values of this*/
+    /*           table. Any changes to this table requires examination of the   */
+    /*           values assigned to these constants.                            */
+    /*                                                                          */
+    /*       EDBG_FIRST_ISTEP_NUM   = 0                                         */
+    /*       EDBG_LAST_ISTEP_NUM    = 21                                        */
+    /*       EDBG_INVALID_ISTEP_NUM = 0xFFFF                                    */
+    /*       EDBG_INVALID_POSITION  = 0xFFFF                                     */
+    /****************************************************************************/
+
+//major | minor |                          istep name   |                     destination   | 
+//number| number|                                       |                                   |
+  { 0,   2,                                   "poweron",                   EDBG_ISTEP_NOOP},
```

Comment:
Need updates in istep numbering
0.1 - poweron
0.2 startipl
0.6 set_ref_clock
0.7 proc_clocktest
0.8	proc_prep_ipl
0.11	proc_select_boot_master
0.13	sbe_config_update
0.14	sbe_start

Pleasew refer IPL flow document